tests/test_pypuck.py

```diff
@@ -61,5 +61,23 @@ def test_player_stats_bad(start_date='2020-02-28', end_date='2019-10-02'):
 	except:
 		# This error is expected
 		pass
+    
 
 
+def test_team_stats_good(start_season='20192020', end_season='20192020'):
+	"""
+	Test function to check proper inputs and returns.
+
+	Keyword Arguments:
+		start_date {str} -- start_date to query (default: {'20192020'})
+		end_date {str} -- end_date to query (default: {'20192020'})
+
+	Raises:
+		TypeError: A message if return type is wrong (wrong data type).
+		ValueError: A message if return value is wrong (not enough data).
```

Comment:
@ManishPJoshi This docstring was just copied from my function. Please add in an appropriate docstring for your function.

```suggestion
	Function to test that the `team_stats` function appropriately returns a pd.DataFrame object and that the resulting dataframe has the queried data. 

	Keyword Arguments:
		start_season {str} -- start_season to query (default: {'20192020'})
		end_season {str} -- end_season to query (default: {'20192020'})

	Raises:
		ValueError: A message if return value is wrong (empty return).
```